addon/content/reducer.js

```diff
@@ -107,7 +128,167 @@ const attachmentActions = {
   },
 };
 
+// TODO: Once the WebExtension parts work themselves out a bit more,
+// determine if this is worth sharing via a shared module with the background
+// scripts, or if it doesn't need it.
+const kScrollUnreadOrLast = 0;
+const kScrollSelected = 1;
```

Comment:
Are these supposed to be Booleans or integers?

Reply:
Integers. I've never been quite happy with these though, hopefully one day we can figure out how to simplify the scroll mode handling.